dashboard/utils.py

```diff
@@ -219,6 +219,16 @@ def get_course_paid_count(self, course_key):
             order__status=Order.FULFILLED, order__user=self.user, course_key=course_key
         ).values('order_id').distinct().count()
 
+    def get_payments_count_for_course(self, course):
+        """
+        Get the total count of payments for given course
+        Args:
+           course (courses.models.Course): a course
+        Returns:
+            int: count of paid course runs
+        """
+        return sum([self.get_course_paid_count(course_run.edx_course_key) for course_run in course.courserun_set.all()])
```

Comment:
Suggest `course.courserun_set.only("edx_course_key")` to make this a little more efficient.